index.html

```diff
@@ -70,37 +70,38 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
         <div class="social">
             <ul class="social-icons">
                 <li>
-                    <a href="#"><img src="./images/github.png" alt=""/></a>
+                    <a href="#"><img src="./images/github.png" alt="github icon"/></a>
```

Comment:
- [ ] Great job adding the alt text for the icons images 👏  but for functional images, the alt text should convey its purpose, you can read more about it [here](https://www.w3.org/WAI/fundamentals/accessibility-principles/#alternatives). For instance, instead of having `github icon`, you could have `github link`.
- [ ] Beware that this change should be applied to all icons, both here and in the footer.